exchanges/bittrex/bittrex_ws_orderbook.go

```diff
@@ -93,7 +93,9 @@ func (b *Bittrex) UpdateLocalOBBuffer(update *OrderbookUpdateMessage) (bool, err
 
 	err = b.applyBufferUpdate(currencyPair)
 	if err != nil {
-		b.flushAndCleanup(currencyPair)
+		if b.Verbose {
+			log.Debugf(log.WebsocketMgr, "UpdateLocalOBBuffer: Could not apply buffer update\n")
```

Comment:
In line with what has already mentioned, it's also nice to prefix this and the below errors/debug/info output with the exchange name for when a person has many exchanges enabled to see what exchange is having this particular issue:

```suggestion
			log.Errorf(log.WebsocketMgr, "%s websocket UpdateLocalOBBuffer: Could not apply buffer update\n", b.Name)
```

Reply:
Good point, thanks!
(I also removed the verbose check as it's a non-trivial error)